concepts/basics/about.md

```diff
@@ -0,0 +1,108 @@
+# About
+
+[Lua](https://lua.org) is a simple yet powerful, efficient, lightweight, embeddable scripting language. It is designed, implemented, and maintained at [PUC-RIO](https://www.puc-rio.br/) in Brazil since 1993 by the team of Roberto Ierusalimschy, Waldemar Celes and Luiz Henrique de Figueiredo.
+
+## Comments
+
+Singleline comments in Lua start with `--` and multiline comments start with `--[[` and end with `]]`.
+```lua
+-- This is a single line comment.
+
+--[[This is
+a multiline
+comment
+]]
+```
+
+## Variables
+
+There are three types of variables in Lua: global variables, local variables and table fields. To keep things simple only local variables are explained here. By default all variables are global so we have to declare local variables as such with the keyword `local`.
+```lua
+local x
+```
+
+A value can be assigned to a variable by using the `=` operator.
+```lua
+local x = 1
+```
+
+The scope of a `local` variable is limited to the block where it is declared. A block can be the body of a control structure, the body of a function or a file or string where the variable is declared.
+```lua
+local x = 1
+
+if x == 1 then
+    local x = 2
+    print(x) -- prints 2
+end
```

Comment:
```suggestion
if x == 1 then
  local x = 2
  print(x) -- prints 2
end
```

We use 2 line indent in the exercises so we should probably use 2 line indents here as well